Demos/benchmarks/setup.py

```diff
@@ -9,5 +9,5 @@
   name = 'benchmarks',
   ext_modules = cythonize("*.py", language_level=3, annotate=True,
                           compiler_directives=directives,
-                          exclude="setup.py"),
+                          exclude=["setup.py","chaos.py"]),
```

Comment:
You might be using the wrong Cython version. Could you revert your change in the Makefile where you are referring to the "cython" command rather than using the relative path to the base directory?